GeometryCore/src/AABB.cpp

```diff
@@ -15,17 +15,26 @@ namespace geom_utils
     }
 
     bool AABB2D::contains(const FPoint2D& p) const
-    { 
-        return{}; 
+    {
+        if (p.x <= pmax.x && p.y <= pmax.y && p.x >= pmin.x && p.y >= pmin.y)  return true;
+        else return false;
     }
 
     void AABB2D::calculate(const Polygon& poly)
     {
-
+        auto maxX = std::max_element(poly.begin(), poly.end(), [](auto a, auto b) {return b.x > a.x; }) - poly.begin();
```

Comment:
What will happen if input poly is empty Polygon?

Reply:
Didn't think about it. Need to chek? Is it all?

Reply:
U can check and u will see) Please fix it